runtime/development/src/lib.rs

```diff
@@ -1696,6 +1704,81 @@ impl pallet_liquidity_rewards::Config for Runtime {
 	type WeightInfo = ();
 }
 
+const WEIGHT_PER_GAS: u64 = 20_000;
```

Comment:
This is copied directly from the Frontier template project - TODO is to determine if this is a good value (large enough to be safe without wasting block time)

Reply:
It may be a good idea to how Moonbeam does this